core/store/models/log_events.go

```diff
@@ -423,10 +426,26 @@ func (parseRunLog20190207withoutIndexes) parseJSON(log Log) (JSON, error) {
 		return JSON{}, err
 	}
 
+	// 8 bytes worth of data versions is plenty.
```

Comment:
Presumably this is defined by the contract, no?

Reply:
good question - yep its 256 versions, ill adjust

Reply:
well actually maybe I'll just comment, for each new version we'd add support in the switch any unsupported version would fail anyways

Reply:
256 version is 8 bits not 8 bytes.

I dont think the comment is necessary tbh

Reply:
256 is the hard coded number of versions in the contract. It made sense to pull 8 bytes instead of 1 in case we we're to increase that in the contract